### PR TITLE
Don't interfere with networking during unittests

### PR DESCRIPTION
**Pull request by AndrejMitrovic**

### CODE REVIEW

File: source/agora/test/Base.d [hunk 1]
```diff
@@ -202,7 +203,7 @@ public NetworkT makeTestNetwork (NetworkT : TestNetwork)
             c => c.key_pair.address.toString()).array);
         foreach (idx, ref conf; configs)
         {
-            net.apis ~= net.createNewNode(node_configs[idx].key_pair.address.toString(), conf);
+            net.apis[node_configs[idx].key_pair.address] = net.createNewNode(node_configs[idx].key_pair.address.toString(), conf);
```
Comment (Geod24) [inline source/agora/test/Base.d]:
```d
const address = node_configs[idx].key_pair.address;
net.apis[address] = net.createNewNode(address.toString(), conf);
```

Comment (AndrejMitrovic) [reply source/agora/test/Base.d]: ah it's a shame you didn't do it as a suggestion

Comment (Geod24) [reply source/agora/test/Base.d]: What's this fancy new tech you're talking about ?

**Comment by AndrejMitrovic** [comment]
Fixed code styling.

File: source/agora/test/Base.d [hunk 2]
```diff
@@ -202,7 +203,9 @@ public NetworkT makeTestNetwork (NetworkT : TestNetwork)
             c => c.key_pair.address.toString()).array);
         foreach (idx, ref conf; configs)
         {
-            net.apis ~= net.createNewNode(node_configs[idx].key_pair.address.toString(), conf);
+            const address = node_configs[idx].key_pair.address;
+            net.apis[address] = net.createNewNode(
+                node_configs[idx].key_pair.address.toString(), conf);
```
Comment (Geod24) [inline source/agora/test/Base.d]: Ahem

**Comment by AndrejMitrovic** [comment]
Fixed!

**Review by Geod24 — APPROVED** [review]
`auto-merge` toggled on

**Comment by MichaelKim20** [comment]
It is difficult to distinguish between classes agora.test.Network.Network and agora.node.Network.Network. 
I think it would be better to change agora.test.Network.Network to a different name. Please review it.

**Comment by Geod24** [comment]
@AndrejMitrovic : Any idea ? We could go with `UnittestNetwork` somewhere.

**Comment by AndrejMitrovic** [comment]
> It is difficult to distinguish between classes agora.test.Network.Network and agora.node.Network.Network.
> I think it would be better to change agora.test.Network.Network to a different name. Please review it.

Yes I think so too. @Geod24 I initially changed it in one of my previous PRs but your changes reverted this. There's a number of ways we could name this, I'll think of something.

**Comment by Geod24** [comment]
Let's merge this and change the name in another PR. Thanks for the suggestion @MukeunKim .

**Pull request merged**